src/test/resources/Features/OCD-1776-SEDTweaks.feature

```diff
@@ -1,33 +1,31 @@
 @Regression
-Feature: SED Display in product details
+Feature: OCD-1776 - SED Display in product details
   This test is to verify text changes and other display updates to SED section
 
-  Scenario: Verify SED details
-    Given I am on the Details page of Listing "8338"
+  Scenario Outline: Verify SED details
```

Comment:
Why is this one an Outline? There's only one example, and I don't think it's likely we'd need to check multiple Listings to verify this Feature